Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -3158,7 +3162,7 @@ - (void)endRecordingVoiceMemo
     self.audioRecorder = nil;
 
     NSString *filename = [NSLocalizedString(@"VOICE_MESSAGE_FILE_NAME", @"Filename for voice messages.")
-        stringByAppendingPathExtension:[MIMETypeUtil fileExtensionForUTIType:(NSString *)kUTTypeMPEG4Audio]];
+        stringByAppendingPathExtension:@".m4a"];
```

Comment:
I'm pretty sure we should .m4a, not .aac.  .mp4 is the container format and .m4a is the right extension for a .mp4 that contains .aac (as opposed to video).  But I think Android is using .aac, which indicates a raw aac-encoded a bitstream. I'm not sure, but I doubt that's what Android is sending. I'm going to follow up with Moxie.